bsconfig.json

```diff
@@ -1,19 +1,18 @@
 {
   "name": "bucklescript-tea",
-  "version": "0.1.1",
+  "version": "0.1.2",
   "bsc-flags": [
     "-w",
     "-40",
-    "-bs-sort-imports",
-    "-bin-annot",
     "-bs-cross-module-opt"
   ],
   "package-specs": [
     "commonjs"
   ],
   "sources": [
     {
-      "dir": "src"
+      "dir": "src",
+      "public" : "all"
```

Comment:
This public option will only expose the given files to be used by dependents?  Can these public interfaces still use non-public interfaces?  And what will they (both public and non-public) do in the case of a name conflict with another library?